gdal_subsetter/utilities.py

```diff
@@ -86,6 +88,31 @@ def is_geotiff(file_name: str) -> bool:
     return gdalinfo_lines[0] == "Driver: GTiff/GeoTIFF"
 
 
+def get_geotiff_variables(filename: str) -> dict[str, str]:
+    """Extract band to standard name mapping from a GeoTIFF.
+
+    The keys of the mapping are "BandN" where "N" is the one-based band number.
+    The values are either:
+
+    * The standard_name as retrieved from the band metadata.
+    * "BandN", if there is no standard_name.
+
+    """
```

Comment:
I think I like the new dictionary comprehension in place of a for loop with an if/else inside. But if it isn't as readable, then let me know.